test-automation/config.properties

```diff
@@ -0,0 +1,2 @@
+digitalFormsBaseUrl = https://digitalforms-api-totdho-dev.pathfinder.gov.bc.ca/digitalforms/
+
```

Comment:
I think this should be from a environment variable too. It would make sense to add a env variable template that can be used if someone else wants to run it locally. Example: https://github.com/bcgov/jag-rsbc-digital-forms-vips-integration/blob/master/.env.template